.github/test-basic-pwc-failed-only.yml

```diff
@@ -30,7 +30,7 @@ jobs:
     name: "Playwright Tests - check failed"
     timeout-minutes: 60
     runs-on: ubuntu-22.04
-    container: mcr.microsoft.com/playwright:latest
+    container: mcr.microsoft.com/playwright:v1.49.0-jammy
```

Comment:
the `latest` tag doesn't seem to get updated anymore

Reply:
yes, did the same on currents-playwright repo.